packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/callbacks/Callback.kt

```diff
@@ -0,0 +1,5 @@
+package expo.modules.kotlin.callbacks
+
+interface Callback<T> {
+  fun invoke(arg: T)
+}
```

Comment:
I bet you know about it, but I still will give it a try:
 https://kotlinlang.org/docs/fun-interfaces.html
```suggestion
fun interface Callback<T> {
  fun invoke(arg: T)
}
```